cmd/prometheus-multi-tenant-proxy/main.go

```diff
@@ -40,13 +40,17 @@ func main() {
 					Name:  "unprotected-endpoints",
 					Usage: "Unprotected endpoints (mostly for live/readiness probes)",
 					Value: cli.NewStringSlice("/-/healthy", "/-/ready"),
+				}, &cli.StringFlag{
+					Name:  "auth-type",
+					Usage: "Auth mechanism: one of 'basic' or 'jwt'",
+					Value: "authn.yaml",
```

Comment:
`auth-type` flag default value must be `basic` instead of `authn.yaml`

Reply:
God, thank you for spotting this silly copy-paste mistake!